packages/web3-eth/src/rpc_methods.ts

```diff
@@ -458,13 +458,13 @@ export async function submitWork(
 	requestManager: Web3RequestManager,
 	nonce: HexString8Bytes,
 	seedHash: HexString32Bytes,
```

Comment:
As per the spec, this parameter is `hash` not `seedHash`. 

https://github.com/ethereum/execution-apis/blob/68cf607f01bd41aecd0aaafd2bfc01928e188cc2/src/eth/mining.json#L63

Reply:
fixed